java/compaction-core/src/test/java/sleeper/compaction/job/CompactionJobRunTest.java

```diff
@@ -0,0 +1,193 @@
+
+        // Then
+        CompactionJobRun jobRun1 = status.getLatestJobRun();
+
+        assertThat(jobRun1.getStartedStatus())
+                .isEqualTo(started2);
+        assertThat(jobRun1.getFinishedStatus())
+                .isEqualTo(finished2);
+    }
+
+    @Test
+    public void shouldBuildCompactionJobStatusWithMultipleJobRunsSameTask() {
+        // Given
+        CompactionJobCreatedStatus created1 = CompactionJobCreatedStatus.builder()
+                .updateTime(Instant.parse("2022-09-23T09:23:00.012Z"))
+                .partitionId("partition1").childPartitionIds(null)
+                .inputFilesCount(11)
+                .build();
+        CompactionJobStartedStatus started1 = CompactionJobStartedStatus.updateAndStartTime(
+                Instant.parse("2022-09-23T09:23:30.012Z"),
+                Instant.parse("2022-09-23T09:23:30.001Z"));
+        CompactionJobFinishedStatus finished1 = CompactionJobFinishedStatus.updateTimeAndSummary(
+                Instant.parse("2022-09-23T09:24:00.012Z"),
+                new CompactionJobSummary(new CompactionJobRecordsProcessed(450L, 300L),
+                        Instant.parse("2022-09-23T09:23:30.001Z"),
+                        Instant.parse("2022-09-23T09:24:00.001Z")));
+        CompactionJobStartedStatus started2 = CompactionJobStartedStatus.updateAndStartTime(
+                Instant.parse("2022-09-24T09:23:30.012Z"),
+                Instant.parse("2022-09-24T09:23:30.001Z"));
+        CompactionJobFinishedStatus finished2 = CompactionJobFinishedStatus.updateTimeAndSummary(
+                Instant.parse("2022-09-24T09:24:00.012Z"),
+                new CompactionJobSummary(new CompactionJobRecordsProcessed(450L, 300L),
+                        Instant.parse("2022-09-24T09:23:30.001Z"),
+                        Instant.parse("2022-09-24T09:24:00.001Z")));
+
+        // When
+        CompactionJobStatus status = new CompactionJobStatusesBuilder()
+                .jobCreated("job1", created1)
+                .jobStarted("job1", started1, DEFAULT_TASK_ID_1)
+                .jobFinished("job1", finished1, DEFAULT_TASK_ID_1)
+                .jobStarted("job1", started2, DEFAULT_TASK_ID_1)
+                .jobFinished("job1", finished2, DEFAULT_TASK_ID_1)
+                .build().get(0);
+
+        // Then
+        assertThat(status.getJobRuns())
+                .contains(
+                        CompactionJobRun.finished(DEFAULT_TASK_ID_1, started1, finished1),
+                        CompactionJobRun.finished(DEFAULT_TASK_ID_1, started2, finished2));
+    }
+
+    @Test
+    public void shouldBuildCompactionJobStatusWithMultipleJobRunsDifferentTasks() {
+        // Given
+        CompactionJobCreatedStatus created1 = CompactionJobCreatedStatus.builder()
+                .updateTime(Instant.parse("2022-09-23T09:23:00.012Z"))
+                .partitionId("partition1").childPartitionIds(null)
+                .inputFilesCount(11)
+                .build();
```

Comment:
There's only one created status. It might be clearer if this variable name wasn't numbered, to distinguish it from the start & finish statuses.